app/views/entities/new.html copy.erb

```diff
@@ -0,0 +1,43 @@
+<main class="body-container">
+  <section class="container">
+    <%= form_for [current_user, @group, @entity] do |f| %>
+      p "@entity"
+      p @entity
+      <% if @entity.errors.any? %>
+        <div id="error_explanation">
+          <h2><%= pluralize(@entity.errors.count, "error") %> prohibited this transacction from being saved:</h2>
+          <ul>
+            <% @entity.errors.full_messages.each do |message| %>
+              <li class="alert-danger"><%= message %></li>
+            <% end %>
+          </ul>
```

Comment:
### Add a new transaction page

- [OPTIONAL]  According to the project requirements, When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category and not the Home Page. Kindly correct this as well. ✍🏻 
- [OPTIONAL] It's one of the project requirements that The user fills out a form to create a new transaction with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one). 

![image](https://user-images.githubusercontent.com/79058364/236657470-343caf8a-7d94-4e0c-8bba-ae4b6a6858dd.png)


- [OPTIONAL] Kindly add validation to the categories input select field to prevent any user to submit the form without selecting any category👍🏻